src/Omnipay/Common/Helper.php

```diff
@@ -33,6 +34,26 @@ function ($match) {
     }
 
     /**
+     * Convert strings with underscores to be all lowercase before camelCase is preformed.
+     *
+     * @param  string $str The input string
+     * @return string The output string
+     */
+    protected static function convertToLowercase($str)
```

Comment:
Okay, fair enough. It just looked a bit strange effectively doing: `if (strpos($str, '_') !== 0) $str = strtolower($str);`

So `This_String` becomes `this_string`, but `ThisString` remains unchanged.